packages/editor/src/functions/EditorControlFunctions.ts

```diff
@@ -221,6 +221,7 @@ const createObjectFromSceneElement = (
   beforeEntity = null as Entity | null,
   updateSelection = true
 ) => {
+  parentEntity = parentEntity ?? getState(SceneState).sceneEntity
```

Comment:
Why isn't the default parameter catching this?

Reply:
In case `addMediaNode()`, `parent` is explicitly passed as `null`: https://github.com/EtherealEngine/etherealengine/blob/53e4d7199156c0cbc700e96bf6b69e89803c02b0/packages/editor/src/functions/addMediaNode.ts#L61-L70

Reply:
ok, wherever we are passing as null we should pass in as undefined instead, to avoid this redundant double nullish coalesce